### PR TITLE
feat(pci): log to customer - tracking

### PR DESCRIPTION
**Pull request by dectotam**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/log-to-customer
| Bug fix?         |  no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat/MANAGER-13920
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

implement tracking in log to customer module and pci / audit-logs

### CODE REVIEW

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import { DOC_URL } from './constants';
+import { DOC_URL, TRACKING_TABS } from './constants';
 
 export default class KubernetesCtrl {
   /* @ngInject */
-  constructor($scope) {
+  constructor($scope, $state) {
     this.$scope = $scope;
+    this.$state = $state;
```
Comment (JacquesLarique) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js:7]: Why did you add $state here, it does not seem to be used

File: packages/manager/modules/log-to-customer/src/live-tail/template.html [hunk 2]
```diff
@@ -38,6 +38,8 @@
             go-to-listing-page="$ctrl.goToListingPage"
             go-to-stream-listing-page="$ctrl.goToStreamListingPage"
             set-error-message="$ctrl.setErrorMessage"
+            track-click="$ctrl.trackClick"
+            tracking-hits="$ctrl.trackingHits"
```
Comment (darsene) [inline packages/manager/modules/log-to-customer/src/live-tail/template.html:42]:
```suggestion
            data-track-click="$ctrl.trackClick"
            data-tracking-hits="$ctrl.trackingHits"
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.html [hunk 3]
```diff
@@ -5,6 +5,8 @@
     go-to-listing-page="$ctrl.goToListingPage"
     log-kinds-list="$ctrl.logKindsList"
     kind-init-value="$ctrl.kind"
+    track-click="$ctrl.trackClick"
+    tracking-hits="$ctrl.LOG_TRACKING_HITS"
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/audit-logs.html:9]:
```suggestion
    data-track-click="$ctrl.trackClick"
    data-tracking-hits="$ctrl.LOG_TRACKING_HITS"
```

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js (hunk 1 above)
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js:7]: Why do you need `$state` ?

File: packages/manager/modules/log-to-customer/src/list/component.js [hunk 4]
```diff
@@ -13,5 +13,7 @@ export default {
     goBack: '<',
     title: '@',
     description: '@',
+    trackingHits: '<',
+    trackClick: '<',
```
Comment (darsene) [inline packages/manager/modules/log-to-customer/src/list/component.js:17]: Where is defined the root `trackClick` function ? I don't see it in your pull request ?

**Review by darsene — CHANGES_REQUESTED** [review]

File: packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js (hunk 1 above)
Comment (dectotam) [reply packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js:7]: I used it during the dev process but it is indeed useless in the end, I remove it.

File: packages/manager/modules/log-to-customer/src/list/component.js (hunk 4 above)
Comment (dectotam) [reply packages/manager/modules/log-to-customer/src/list/component.js:17]: the function is passed through the parent (see audit-logs.html)

**Review by JacquesLarique — APPROVED** [review]

File: packages/manager/modules/log-to-customer/src/list/component.js (hunk 4 above)
Comment (darsene) [reply packages/manager/modules/log-to-customer/src/list/component.js:17]: Which file exactly ?

**Review by darsene — APPROVED** [review]
